agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptionsParser.java

```diff
@@ -92,6 +92,13 @@ private void handleOption(AgentOptions options, String optionPart) throws AgentO
 		Pair<String, String> keyAndValue = parseOption(optionPart);
 		String key = keyAndValue.getFirst();
 		String value = keyAndValue.getSecond();
+		if (key.startsWith("debug")) {
+			options.debugLogging = true;
+			if (!value.isEmpty() && !value.equalsIgnoreCase("true")) {
+				options.debugLogDirectory = Paths.get(value);
+			}
+			return;
+		}
```

Comment:
Can you extract this to a own method to remove the Teamscale finding and to make it look cleaner here, especially with the nesting if